setup.py

```diff
@@ -41,6 +41,13 @@
 from scripts.package_helpers import get_packages_with_tests
 from scripts.version import get_onedal_shared_libs, get_onedal_version
 
+USE_ABS_RPATH = False
+abs_rpath_args = ["abs-rpath", "-abs-rpath", "--abs-rpath"]
```

Comment:
Why three types of prefix? Double is closest to a standard.

Reply:
Modified to accept only a double-dashed argument.